cpp/include/Ice/CtrlCHandler.h

```diff
@@ -0,0 +1,18 @@
+//
+// Copyright (c) ZeroC, Inc. All rights reserved.
+//
+
+#ifndef ICE_CTRL_C_HANDLER_H
+#define ICE_CTRL_C_HANDLER_H
+
+#include <IceUtil/CtrlCHandler.h>
+
+namespace Ice
+{
+
+using CtrlCHandlerCallback = IceUtil::CtrlCHandlerCallback;
+using CtrlCHandler = IceUtil::CtrlCHandler;
```

Comment:
These aliases were previously defined in Application.h

Reply:
We should move CtrlCHandler to namespace Ice.

Reply:
I didn't because is still used by Slice compilers.